tests/Compiler/ComponentTagCompilerTest.php

```diff
@@ -22,6 +22,27 @@ public function testShouldCompileTwigVariables(): void
         $this->assertSame('{% embed "@alias/alert.twig" with { props: {\'variable\': value,\'variableWithoutQuotes\': value} } %}{% endembed %}', $compiler->compile());
     }
 
+    public function testShouldCompileTwigVariablesWithTwigCommentSelfClosingTag(): void
+    {
+        $compiler = new ComponentTagCompiler('<Alert variable="{{value}}" {# This is comment #} variableWithoutQuotes={{value}} />', 'alias');
```

Comment:
Why can't something like this be used? For example if I want to remove something during debugging.

```
<Alert {# variable="{{value}}" #} variableWithoutQuotes={{value}} />
```

... but this won't be matched by the regex. Is this intentional?

Reply:
Added into test matrix also with empty comment `{# #}` advised by Lukáš Zemek. Both now work with your solution using negative lookahead. Thanks.